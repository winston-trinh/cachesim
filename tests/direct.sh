./csim -S 8 -K 1 -B 1 -p LRU -t traces/simple_direct.trace
./csim -S 4 -K 1 -B 1 -p LRU -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 1 -p LRU -t traces/simple_direct.trace
./csim -S 1 -K 1 -B 4 -p LRU -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 4 -p LRU -t traces/simple_direct.trace
./csim -S 1 -K 1 -B 8 -p LRU -t traces/simple_direct.trace
./csim -S 4 -K 1 -B 2 -p LRU -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 2 -p LRU -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 2 -p LRU -t traces/yi2.trace
./csim -S 4 -K 1 -B 16 -p LRU -t traces/dave.trace
./csim -S 4 -K 1 -B 8 -p LRU -t traces/trans.trace
./csim -S 32 -K 1 -B 32 -p LRU -t traces/trans.trace
./csim -S 32 -K 1 -B 32 -p LRU -t traces/long.trace
./csim -S 8 -K 1 -B 1 -p FIFO -t traces/simple_direct.trace
./csim -S 4 -K 1 -B 1 -p FIFO -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 1 -p FIFO -t traces/simple_direct.trace
./csim -S 1 -K 1 -B 4 -p FIFO -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 4 -p FIFO -t traces/simple_direct.trace
./csim -S 1 -K 1 -B 8 -p FIFO -t traces/simple_direct.trace
./csim -S 4 -K 1 -B 2 -p FIFO -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 2 -p FIFO -t traces/simple_direct.trace
./csim -S 2 -K 1 -B 2 -p FIFO -t traces/yi2.trace
./csim -S 4 -K 1 -B 16 -p FIFO -t traces/dave.trace
./csim -S 4 -K 1 -B 8 -p FIFO -t traces/trans.trace
./csim -S 32 -K 1 -B 32 -p FIFO -t traces/trans.trace
./csim -S 32 -K 1 -B 32 -p FIFO -t traces/long.trace
