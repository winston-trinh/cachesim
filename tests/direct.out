hits:3 misses:3 evictions:0
hits:2 misses:4 evictions:2
hits:0 misses:6 evictions:5
hits:3 misses:3 evictions:2
hits:4 misses:2 evictions:0
hits:5 misses:1 evictions:0
hits:3 misses:3 evictions:0
hits:2 misses:4 evictions:2
hits:9 misses:8 evictions:6
hits:2 misses:3 evictions:1
hits:167 misses:71 evictions:67
hits:231 misses:7 evictions:0
hits:265189 misses:21775 evictions:21743
hits:3 misses:3 evictions:0
hits:2 misses:4 evictions:2
hits:0 misses:6 evictions:5
hits:3 misses:3 evictions:2
hits:4 misses:2 evictions:0
hits:5 misses:1 evictions:0
hits:3 misses:3 evictions:0
hits:2 misses:4 evictions:2
hits:9 misses:8 evictions:6
hits:2 misses:3 evictions:1
hits:167 misses:71 evictions:67
hits:231 misses:7 evictions:0
hits:265189 misses:21775 evictions:21743
